docs/modules/ROOT/pages/index.adoc

```diff
@@ -837,7 +837,7 @@ In this scenario, the _<<_identity_assertion_consumer,identity assertion consume
 
 . Is there a direct trust relationship with the _<<_named_actor,named actor>>?_ If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *trusted.*
 . Is there a transitive trust relationship with the _<<_named_actor,named actor>>_ via its _credential issuer?_ (In other words, does the _<<_identity_assertion_consumer,identity assertion consumer>>_ trust the _credential issuer_ to issue valid signature credentials?)
-.. If so, has the _credential issuer_ issued a revocation for the _<<_named_actor,named actor’s>>_ credential? If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
+.. If so, has the _credential issuer_ issued a revocation for the _<<_named_actor,named actor’s>>_ credential that was effective at the time the *<<_identity_assertion,identity assertion>>* was signed? If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*
```

Comment:
I disagree.  If you trust that the credential issuer was doing their job and if the credential was valid at the time of signature, then the assertion should be trusted even if the credential was subsequently revoked. (If the credential was revoked at the time of signature, then it should probably not have been allowed to create the signature.) 
Credential revocation is part of the credential lifecycle. If you change your address and it's on your credential, that credential will be revoked (and a new one issued). If your credential was issued by your state (e.g., California ), and you move states and so your California credential is revoked after you get a Washington credential, you original identity assertion should still be trusted.
I think it may make sense to "trusted, but since revoked", so a consumer could investigate further if they wanted.

Reply:
Or is this intended to say, "If so, had the _credential issuer_ revoked the _<<_named_actor,named actor’s>>_ credential before the time the *<<_identity_assertion,identity assertion>>* was signed? If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *untrusted.*

Reply:
@dbigsby I think we need to interrogate the _cause_ for revocation a bit more to make that determination. In the change-of-address scenario you are positing, it feels entirely reasonable to accept a credential if it was believed to be valid at time of signature. However, in a credential compromise scenario, you might want or need to backdate a revocation and thus invalidate signatures created using the compromised credentials.

Can we find a way to address both situations?